### PR TITLE
Add Muscle node description

### PR DESCRIPTION
**Pull request by stefaniapedrazzi**
Related to https://github.com/omichel/webots/pull/4617.

https://www.cyberbotics.com/doc/reference/muscle?version=add-muscle-node

### CODE REVIEW

**Comment by stefaniapedrazzi** [comment]
:+1:

**Comment by stefaniapedrazzi** [comment]
How do you expect it to behave if the motor is controlled in position?

**Comment by omichel** [comment]
I meant the force which can be measured from the ODE feedback function.

**Comment by stefaniapedrazzi** [comment]
Ok, but in kinematic mode there won't be any feedback.

**Comment by stefaniapedrazzi** [comment]
:+1:

File: reference/muscle.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  field MFColor  colors      []            # idle (0), contracting (1), and relaxing(2) state colors
+  field SFBool   castShadows TRUE
+  field SFBool   visible     TRUE
+}
+```
+
+### Description
+
+A [Muscle](#muscle) node can be used to graphically display the contraction of an artificial muscle implemented using [Joint](joint.md) and [Motor](motor.md) nodes.
+The artificial muscle is represented using a spheroid where the symmetry axis is the vector between the joint's closest upper [Transform](transform.md) node (or `startOffset` position) and the `endPoint` [Solid](solid.md) node (or `endOffset` position).
```
Comment (omichel) [inline reference/muscle.md]: I would remove the "(or `start/endOffset` position)" as it looks confusing and anyhow it is better explained below.

**Comment by stefaniapedrazzi** [comment]
:+1:

**Pull request merged**